app/src/main/java/com/example/coronadiagnosticapp/data/db/entity/HealthResult.kt

```diff
@@ -2,11 +2,14 @@ package com.example.coronadiagnosticapp.data.db.entity
 
 import androidx.room.Entity
 import androidx.room.PrimaryKey
+import com.example.coronadiagnosticapp.ui.fragments.oxymeter.OxymeterData
 import java.util.*
 
 @Entity(tableName = "health_table")
 data class HealthResult(
-    @PrimaryKey val date: Date,
+    @PrimaryKey
+    val date: Date,
```

Comment:
What happens if there are 2 measurements in one day?

Reply:
I think it will just send with the current date 
Date obj contains time as well :)